Sources/ParseSwift/Protocols/ParseObjectMutable.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ParseObjectMutable.swift
+//  ParseSwift
+//
+//  Created by Damian Van de Kauter on 30/10/2021.
+//  Copyright © 2021 Parse Community. All rights reserved.
+//
+
+import Foundation
+
+/**
+ The `ParseObjectMutable` protocol adds an empty version of the respective object.
+ This can be used when you only need to update a subset of the fields of an object
+ as oppose to updating every field of an object.
+ Using an empty object and updating a subset of the fields reduces the amount of data
+ sent between client and server when using `save` and `saveAll`
+ to update objects.
+ 
+ **Example use case:**
+ ````
+ var user = User.current?.mutable
```

Comment:
The “current” example is good, can you add the `Users` struct to the top of the example so the developer can see where `User` Came from?

Also, can you add a second example using `GameScore`?

Reply:
I would add the whole GameScore example from the playgrounds with the extension: https://github.com/parse-community/Parse-Swift/blob/126f4f6a7ccf7ec91fe3f5acfcedd30ad19bb012/ParseSwift.playground/Pages/1%20-%20Your%20first%20Object.xcplaygroundpage/Contents.swift#L32-L69